lib/babel_compile.js

```diff
@@ -13,6 +15,15 @@ module.exports = function(source, compileOptions, options){
 	];
 	opts.plugins = opts.plugins || [];
 
+	// register plugins not bundled in babel-standalone
+	opts.plugins.forEach(function(plugin) {
+		var name = typeof plugin === "string" ? plugin : plugin[0];
+
+		if (!babel.availablePlugins[name]) {
+			babel.registerPlugin(name, require(getBabelPluginPath(name)));
```

Comment:
Yeah, seems brittle. Since this is more of a steal concern (transpile shouldn't really know about Steal or Loaders or anything like that), I think the solution would be that steal-tools takes care of requiring the plugins before passing them over to transpile. It will have the loader.baseURL and can do as you describe.